docker-compose.yml

```diff
@@ -3,16 +3,24 @@ version: "3.8"
 name: quilibrium
 
 # See sysctl related warning in DOCKER-README.md.
-# Currently Docker cannot be used to run Quilibrium.
+# Host configuration changes are required.
 
 services:
   node:
     build: ./
     image: quilibrium
     restart: unless-stopped
+    deploy:
+      resources:
+        limits:
+          memory: 32G
+        reservations:
+          cpus: 12
+          memory: 16G
```

Comment:
@CassOnMars do these resources look reasonable?

Is there a hard coded limit on number of CPUs?

Reply:
For Dawn, I got out of memory even with 32GB

Reply:
> For Dawn, I got out of memory even with 32GB

With the above config? Trying to determine if 32GB is not enough for Quilibrium and Docker is not allowing more, or Quilibrium keeps taking more that 32GB and the host runs out of memory?

Reply:
> > For Dawn, I got out of memory even with 32GB
> 
> With the above config? Trying to determine if 32GB is not enough for Quilibrium and Docker is not allowing more, or Quilibrium keeps taking more that 32GB and the host runs out of memory?

No, not with the config above. I'm currently running Quilibrium natively using Go. Quilibrium seems to keep taking more than 32GB RAM and the host runs out of memory. Happens after the node has been running for about 4 hours

Reply:
16 is the minimum for Dawn, going OOM is a bug